api/ai/new/interfaces/algorithm_options.py

```diff
@@ -42,6 +48,10 @@ def validate(self):
     def parse_json(_: Dict[str, Any]) -> "RandomAlgorithmOptions":
         return RandomAlgorithmOptions()
 
+    @staticmethod
+    def get_schema() -> Schema:
+        return Schema({"algorithm_type": Const(AlgorithmType.RANDOM.value)})
```

Comment:
it's BRUTAL that we have to put the algorithm type in the schema even though it's not in the dataclass, if we can avoid this it would be sick